File: ParcelamentosAPI/DomainCore/Entities/Orcamento.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DomainCore.Entities
{
    public class Orcamento : BaseEntity
    {
        public int ProdutoId { get; set; }

        public decimal ValorBase { get; set; }

        public decimal JurosMes { get; set; }

        public Produto Produto { get; set; }

        public ICollection<Parcela> Parcelas { get; set; }
    }
}

File: ParcelamentosFront/src/Models/Orcamento.ts
import Produto from './Produto';
import Parcela from './Parcela';

export default class Orcamento 
{
    Id: number;

    ProdutoId: number;

    ValorBase: number;

    JurosMes: number;

    QtdParcelas: number;

    DataCompra: string;

    Produto: Produto;

    Parcelas: Parcela[];
}